 <launch>
 	<arg name="is_sim" default="false"/>
  <arg name="local" default="false"/>

  <machine name="movo1" address="movo1" user="movo" 
               env-loader="/home/movo/env.sh" default="true" unless="$(arg local)"/>
	
	<!--include file="$(find movo_moveit_config)/launch/movo_moveit_planning_execution.launch"/-->
	
	<!-- Start Perception -->
    <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" output="screen">
    	<remap from="/head_camera/depth_registered/points" to="/kinect2/qhd/points"/>
    	<param name="simulated_kinova" value="$(arg is_sim)"/>
    	<param name="use_debug" value="true"/>
      <rosparam command="load" file="$(find movo_demos)/config/simple_grasping.yaml"/>
      <rosparam command="load" file="$(find movo_moveit_config)/config/kinematics.yaml" />
    </node>

    <!-- Pick stuff up -->
    <node name="demo" pkg="movo_demos" type="test_segmentation.py" output="screen" />

</launch>
